cfgov/unprocessed/apps/mmt-my-money-calendar/css/components/calendar.less

```diff
@@ -108,9 +146,76 @@
 }
 
 .calendar-details {
-  margin-top: 1.5rem;
+  border: 1px solid @inactiveMonthColor;
+  border-radius: .5rem;
+  padding: 10px;
 
-  &__events {
+  &__header {
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 1rem;
+    text-align: center;
+  }
+
+  .m-notification {
+    text-align: left;
+
+    &_explanation {
+      font-size: 14px;
+    }
+
+    &_button {
+      background-color: @red;
+      color: #fff;
+      display: block;
+      font-weight: bold;
+      text-align: center;
+      margin-top: 12px;
+      padding: 5px;
+    }
+  }
+
+  .notification-icon {
+    position: absolute;
+  }
+
+  .notification-icon + .m-notification_content {
+    padding-left: 1.5625rem;
+  }
+
+  &__nav-button {
+    font-size: 24px;
+    background: none;
+    border: 0;
+
+    &:first-of-type {
+      margin-right: 10px;
+    }
+
+    &::last-of-type {
```

Comment:
```suggestion
    &:last-of-type {
```

Reply:
This is a pseudoclass, not a pseudoelement. I think 1 colon is right in this case.

Reply:
Nevermind, I read the diff wrong. This is indeed a typo

Reply:
Fixed